config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
+  get 'users/new'
+
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
   resources :posts
+  resources :users
```

Comment:
so we've added this resource to our routes.rb - can you remind me of the command to run in rails to force the creation of the actual routes?

Reply:
rails generate controller [NAME] [a bunch of options?]

if you run rails generate controller -h youll get all the options, good luck understanding it though

Reply:
- running `bin/rails generate controller User new` to generate a new controller for user + default views + some other files > we changed the view to get a form and updated the controller to create new users. it also changed the config/routes file by adding `get ‘users/new’`